docs/Agent/user-guide/man_page.md

```diff
@@ -45,7 +45,7 @@
 This command clears the results directory from `/var/lib/pbench-agent` directory.
 
 **OPTIONS**  
- `-C`, `--config` PATH  
+ `-C`, `--config PATH`  
```

Comment:
The way it was wasn't bad...the thing is, if the user uses `-C`, s/he still needs to provide `PATH`...so we need some way to convey that `-C` and `--config` are synonyms, each requiring a value.

Something like
```
`[-C | --config] <PATH>`
```
or
```
`-C <PATH>`, `--config <PATH>`
```

I've also seen man pages (like `du(1)`) which add text like "Mandatory arguments to long options are mandatory for short options too" and then the option synopses look like `-B, --block-size=SIZE`.

Reply:
Yeah, I was moderately happy with the old "`-C`, `--config` PATH" although something like Webb's more explicit alternation syntax plus a symbolic placeholder would be nicer. Like `[-C | --config] <path>`. But that's a detail, and I can clean this up after I rebase when I add the new options.